ipaserver/plugins/internal.py

```diff
@@ -201,10 +201,10 @@ class i18n_messages(Command):
             "auth_indicator": _("Authentication indicator"),
             "config_tooltip": _("<p>Implicit method (password) will be used if no method is chosen.</p><p><strong>Password + Two-factor:</strong> LDAP and Kerberos allow authentication with either one of the authentication types but Kerberos uses pre-authentication method which requires to use armor ccache.</p><p><strong>RADIUS with another type:</strong> Kerberos always use RADIUS, but LDAP never does. LDAP only recognize the password and two-factor authentication options.</p>"),
             "custom_auth_ind_title": _("Add Custom Authentication Indicator"),
-            "otp": _("OTP"),
+            "otp": _("otp"),
             "type_otp": _("Two factor authentication (password + OTP)"),
             "type_password": _("Password"),
-            "type_radius": _("RADIUS"),
+            "type_radius": _("radius"),
```

Comment:
Changing this shows then invalid lowercase value on IPA Server/Configuration page for auth types and also on user details page.

It would be better to create separate:
```
authindicator_otp: _("otp"),
authindicator_radius _("radius"),
```
And use those for auth indicators checkboxes to avoid such side effects.